### ISSUE TITLE
Doesn't work with SASS or SCSS

### ISSUE THREAD

**Issue by jonasflint** [whitequark/rainbowth#20]
doesn't appear to be working for Sass at all....

this would be a amazing plugin.

**Comment by whitequark** [comment]
You can add sass or scss in settings: https://github.com/whitequark/rainbowth/blob/master/Rainbowth.sublime-settings#L2

**Comment by jonasflint** [comment]
Yeah I've tried that. (on multiple machines and instances of sublime text)  It simply does not work.   Man it would be perfect if it did. Super useful. 

Here is what my setting file looks like: 
http://d.pr/i/QiihBV

**Comment by ItamarShDev** [comment]
![image](https://user-images.githubusercontent.com/953371/34384545-77c53af2-eb25-11e7-95fb-1e1415af9c3f.png)

**Comment by ankushdharkar** [comment]
I was able to get it working for Sass files

I have posted some detailed instructions for setup [here](https://github.com/ankushdharkar/rainbowth/blob/546b0603815ca2be93d9d809c09226b81768fd1a/Setup-Instructions.md)
